interface/eRxStore.php

```diff
@@ -112,8 +114,9 @@ public function getPatientVitalsByPatientId($patientId)
 
         $data = formFetch("form_vitals", $result['id']);
 
-        $weight = number_format($data['weight'] * 0.45359237, 2);
-        $height = number_format(round($data['height'] * 2.54, 1), 2);
+        $measurementUtils = new MeasurementUtils();
+        $weight = number_format((float)$measurementUtils->lbToKg($data['weight']), 2);
+        $height = number_format(round((float)$measurementUtils->inchesToCm($data['height']), 1), 2);
```

Comment:
It seems to me that if we need to wrap our call with `number_format`, we're missing a feature in measurementutils.